packages/rich-text/src/remove.js

```diff
@@ -9,9 +9,9 @@ import { create } from './create';
  * Remove content from a Rich Text value between the given `startIndex` and
  * `endIndex`. Indices are retrieved from the selection if none are provided.
  *
- * @param {Object} value      Value to modify.
- * @param {number} startIndex Start index.
- * @param {number} endIndex   End index.
+ * @param {Object}  value       Value to modify.
```

Comment:
Indentation here is inconsistent across the updated docs.

Reply:
Fixed at 50d18e5